service/agentconf.go

```diff
@@ -256,66 +256,61 @@ func (s *systemdServiceManager) CreateAgentConf(name string, dataDir string) (_
 
 // CopyAgentBinary copies all the tools into the path specified for each agent.
 func (s *systemdServiceManager) CopyAgentBinary(
-	machineAgent string,
-	unitAgents []string,
-	dataDir, toSeries, fromSeries string,
-	jujuVersion version.Number,
+	machineAgent string, unitAgents []string, dataDir, toSeries, fromSeries string, jujuVersion version.Number,
```

Comment:
If we add more args here we should consider passing in a struct instead.

Reply:
Agreed.